environment.yml

```diff
@@ -38,4 +38,4 @@ dependencies:
  - shapely
  - tqdm
  - pip:
-   - dem-stitcher>=2.2.0
+   - dem-stitcher>=2.3.0
```

Comment:
Ideally, this would be moved into the conda dependencies once available. 

If conda-forge/staged-recipes#19470 hasn't been acted on by Monday, I'd post https://gitter.im/conda-forge/conda-forge.github.io asking for a review and that usually gets someone on it.

Reply:
Ok - I just updated the [recipe](https://github.com/conda-forge/staged-recipes/pull/19470/commits/3460bc52ec5efe71ecdb52582d3fd4861fc55ff2) to:

1. update acceptable python version 3.10
2. the PyPI sha
3. dem-stitcher 2.3.0

I will reach out on the gitter on Monday to ask for help. This will ensure the newest release can be used here that will fix this issue.

Reply:
I don't think this PR is urgently pressing for continued processing unless we have products that we need to reprocess over Armenia / Azerbaijan.

Reply:
I posted on the gitter channel.

Reply:
From Joe (we are 3rd in queue of recipes that are passing): https://github.com/conda-forge/staged-recipes/pulls?q=is%3Aopen+is%3Apr+label%3Areview-requested+label%3Apython+sort%3Acreated-asc

Reply:
It's on conda-forge so I am going to resolve this in the next commit.